Requesting OAuth tokens fails

There are currently two incidents, both with this stacktrace:

```
jakarta.ws.rs.ProcessingException: RESTEASY004655: Unable to invoke request: jakarta.ws.rs.ProcessingException: RESTEASY003215: could not find writer for content-type application/json type: io.zeebe.clustertestbench.cloud.oauth.OAuthClient$UserAccountTokenRequest
	at org.jboss.resteasy.client.jaxrs.engines.ManualClosingApacheHttpClient43Engine.invoke(ManualClosingApacheHttpClient43Engine.java:321)
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.invoke(ClientInvocation.java:494)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invokeSync(ClientInvoker.java:152)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invoke(ClientInvoker.java:115)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientProxy.invoke(ClientProxy.java:76)
	at jdk.proxy2/jdk.proxy2.$Proxy26.requestToken(Unknown Source)
	at io.zeebe.clustertestbench.cloud.oauth.OAuthInterceptor.lambda$forUserAccountAuthorization$1(OAuthInterceptor.java:94)
	at io.zeebe.clustertestbench.cloud.oauth.OAuthInterceptor.filter(OAuthInterceptor.java:31)
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.filterRequest(ClientInvocation.java:767)
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.invoke(ClientInvocation.java:491)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invokeSync(ClientInvoker.java:152)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invoke(ClientInvoker.java:115)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientProxy.invoke(ClientProxy.java:76)
	at jdk.proxy2/jdk.proxy2.$Proxy34.listGenerationInfos(Unknown Source)
	at io.zeebe.clustertestbench.handler.CreateGenerationInCamundaCloudHandler.findGenerationInfoByName(CreateGenerationInCamundaCloudHandler.java:96)
	at io.zeebe.clustertestbench.handler.CreateGenerationInCamundaCloudHandler.lookupTemplate(CreateGenerationInCamundaCloudHandler.java:90)
	at io.zeebe.clustertestbench.handler.CreateGenerationInCamundaCloudHandler.handle(CreateGenerationInCamundaCloudHandler.java:34)
	at io.zeebe.clustertestbench.handler.JobHandlerWithEnrichedLogger.handle(JobHandlerWithEnrichedLogger.java:26)
	at io.camunda.zeebe.client.impl.worker.JobRunnableFactory.executeJob(JobRunnableFactory.java:44)
	at io.camunda.zeebe.client.impl.worker.JobRunnableFactory.lambda$create$0(JobRunnableFactory.java:39)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: jakarta.ws.rs.ProcessingException: RESTEASY003215: could not find writer for content-type application/json type: io.zeebe.clustertestbench.cloud.oauth.OAuthClient$UserAccountTokenRequest
	at org.jboss.resteasy.core.interception.jaxrs.ClientWriterInterceptorContext.throwWriterNotFoundException(ClientWriterInterceptorContext.java:50)
	at org.jboss.resteasy.core.interception.jaxrs.AbstractWriterInterceptorContext.getWriter(AbstractWriterInterceptorContext.java:302)
	at org.jboss.resteasy.core.interception.jaxrs.AbstractWriterInterceptorContext.syncProceed(AbstractWriterInterceptorContext.java:240)
	at org.jboss.resteasy.core.interception.jaxrs.AbstractWriterInterceptorContext.proceed(AbstractWriterInterceptorContext.java:224)
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.writeRequestBody(ClientInvocation.java:446)
	at org.jboss.resteasy.client.jaxrs.engines.ManualClosingApacheHttpClient43Engine.writeRequestBodyToOutputStream(ManualClosingApacheHttpClient43Engine.java:625)
	at org.jboss.resteasy.client.jaxrs.engines.ManualClosingApacheHttpClient43Engine.buildEntity(ManualClosingApacheHttpClient43Engine.java:584)
	at org.jboss.resteasy.client.jaxrs.engines.ManualClosingApacheHttpClient43Engine.loadHttpMethod(ManualClosingApacheHttpClient43Engine.java:489)
	at org.jboss.resteasy.client.jaxrs.engines.ManualClosingApacheHttpClient43Engine.invoke(ManualClosingApacheHttpClient43Engine.java:299)
	... 25 more
```

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -124,6 +124,9 @@
                 <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                   <mainClass>io.zeebe.clustertestbench.bootstrap.BootstrapFromEnvVars</mainClass>
                 </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/services/jakarta.ws.rs.ext.Providers</resource>
+                </transformer>
               </transformers>
               <finalName>zeebe-cluster-testbench-uber-jar-with-dependencies</finalName>
             </configuration>

